node/src/chain_spec.rs

```diff
@@ -140,12 +138,11 @@ pub fn local_testnet_config(id: ParaId) -> ChainSpec {
 			)
 		},
 		// Bootnodes
-		// These will be configured using the JSON files in the `res` foler
-		Vec::new(),
+		vec![],
 		// Telemetry
 		None,
 		// Protocol ID
-		Some("canvas-rococo"),
+		Some("canvas-local"),
```

Comment:
What is this good for? Does it need to match with some stuff in the relay chain? The ID set above is `canvas_rococo`. Maybe we should be consistent here?

```suggestion
		Some("canvas-rococo"),
```

Reply:
According to [the docs on the chain spec](https://substrate.dev/docs/en/knowledgebase/integrate/chain-spec):

>nodes will only add peers who use the same `protocolId`.

And from [the rustdoc of `ProtocolId`](https://substrate.dev/rustdocs/latest/sc_network/config/struct.ProtocolId.html):

>Name of a protocol, transmitted on the wire. Should be unique for each chain. 

So I think the distinction between local and Rococo still makes sense.

Reply:
Ahh okay. I was looking at the wrong function.

Reply:
@cmichi  Don't remove the protocol id. It is used to identify different networks.